daprdocs/content/en/developing-ai/dapr-agents/dapr-agents-getting-started.md

```diff
@@ -51,162 +51,326 @@ docker ps
+* The `conversation.openai` component type configures the LLM provider and model.
+* `key` holds the API key used to authenticate with the LLM provider.
+
+Replace `{{OPENAI_API_KEY}}` with your actual API key so the Conversation API can perform chat completion.
+
+With this setup, you can swap models or even providers by editing the component YAML without changing the agent code.
+
+### Conversation memory with a Dapr state store
 
-`openai.yaml`:
+In the agent definition, conversation memory is configured as:
+
+```python
+memory=AgentMemoryConfig(
+  store=ConversationDaprStateMemory(
+      store_name="conversation-statestore",
+      session_id="06-durable-agent-http",
+  )
+),
+```
+
+This tells the agent to store conversation history in a Dapr state store named `conversation-statestore`, under a given `session_id`. The matching Dapr component is `resources/conversation-statestore.yaml`:
 
 ```yaml
 apiVersion: dapr.io/v1alpha1
 kind: Component
 metadata:
-  name: openai
+  name: conversation-statestore
 spec:
-  type: conversation.openai
+  type: state.redis
   version: v1
   metadata:
-  - name: key
-    value: "{{OPENAI_API_KEY}}"
-  - name: model
-    value: gpt-5-2025-08-07
-  - name: temperature
-    value: 1
+    - name: redisHost
+      value: localhost:6379
+    - name: redisPassword
+      value: ""
 ```
 
-This component wires the default `DaprChatClient` to OpenAI via the Conversation API. Replace the `{{OPENAI_API_KEY}}` placeholder with your actual OpenAI key by editing the file directly. This API key is essential for agents to communicate with the LLM, as the default chat client talks to OpenAI-compatible endpoints. If you don't have an API key, you can [create one here](https://platform.openai.com/api-keys). You can also tweak metadata (model, temperature, baseUrl, etc.) to point at compatible OpenAI-style providers.
+* The state store uses Redis to persist conversation turns.
+* The agent reads and writes messages here so the LLM can maintain context across multiple HTTP calls.
 
-### 3. Create the agent with weather tool
+You can browse this state later (for example, with Redis Insight) to see how conversation history is stored.
 
-Create `weather_agent.py`:
+### Durable execution state with a workflow state store
+
+The agent’s durable execution state is configured as:
 
 ```python
-import asyncio
-from dapr_agents import tool, Agent
-from dapr_agents.agents.configs import AgentMemoryConfig
-from dapr_agents.memory import ConversationDaprStateMemory
-from dotenv import load_dotenv
+state=AgentStateConfig(
+  store=StateStoreService(store_name="workflow-statestore"),
+),
+```
 
-load_dotenv()
+This uses a Dapr state store named `workflow-statestore` to persist workflow and agent execution state. The corresponding component is `resources/workflow-statestore.yaml`:
 
+```yaml
+apiVersion: dapr.io/v1alpha1
+kind: Component
+metadata:
+  name: workflow-statestore
```

Comment:
We should use the same as the default names here IMO. So `agent-workflow` (although I do prefer `agent-workflow-statestore` to be more explicit.